packages/web-components/src/components/cbp-footer/cbp-footer.scss

```diff
@@ -22,3 +26,9 @@ cbp-footer {
     padding-top: var(--cbp-space-5x);
   }
 }
+
+@media (min-width: $cbp-breakpoint-md) {
```

Comment:
I ran a quick test and I don't think you need this. The flex context takes care of this on its own and the buttons look exactly the same when flexed with 100% width as with auto.